### ISSUE TITLE
scoring system violating when -x asm5

### ISSUE THREAD

**Issue by wangyugui** [lh3/minimap2#86]
Hi.

scoring system violating when -x asm5.

```
# minimap2 -x asm5 -d ${dnaref}.asm5.mmi ${dnaref}
[ERROR] scoring system violating {-A}+2*({-O}+{-E}) <= 127
# minimap2 --version
2.6-r629-dirty
```
but 'minimap2 -d ${dnaref}.mmi ${dnaref}' works well.

**Comment by lh3** [comment]
The check was implemented to address #79. However, my calculation was wrong. I have temporarily disabled the relevant code. It should work now. Thanks.

**Comment by wangyugui** [comment]
It works well now.
